package.json

```diff
@@ -60,7 +60,8 @@
     "postinstall": "webpack -p",
     "test": "karma start karma.conf.js",
     "test-watch": "karma start karma.conf.js --no-single-run",
-    "validate": "npm ls"
+    "validate": "npm ls",
+    "watch": "webpack --config webpack.config.js --watch --mode development"
```

Comment:
I'm curious, how are you using this?

Reply:
I use this script so the lib is _compiled_ on change and in `jitsi-meet` I just have a symlink to it.